src/jit/lower.cpp

```diff
@@ -1167,6 +1167,11 @@ GenTree* Lowering::NewPutArg(GenTreeCall* call, GenTree* arg, fgArgTabEntry* inf
                 }
             }
 
+#ifdef DEBUG
+            if (comp->verbose)
+                printf("Creating putarg at slot %d, num slots %d\n", info->slotNum, info->numSlots);
+#endif
+
```

Comment:
I will get rid of this.

Reply:
I think this is useful to keep as jitdump output.

Reply:
It is basically already there by looking at the trees output -- it prints this information too.